### ISSUE TITLE
Implement TokBox as a Video & Audio Conferencing solution

### ISSUE THREAD

**Issue by designgurudotorg** [RocketChat/Rocket.Chat#6239]
Guys,

I love how easy Jitsi Meet is to use out of the box but have been brainstorming on how we can improve the audio/video experience with a more reliable HD solution that has a clean (unbranded) interface and can more tightly integrate within RocketChat.

Though I love the promise of our built-in WebRTC implementation its just not reliable and I've been thinking that integrating https://tokbox.com/ makes a lot of sense.  TokBox offers Video (https://tokbox.com/platform/video), Audio (https://tokbox.com/platform/voice) and Screensharing (https://tokbox.com/platform/screen-sharing) Also, they have Android and IOS SDks (https://tokbox.com/platform/mobile) - not sure how that would work with RC's Cordova apps but definitely offers quality AV integration into mobile...

Thoughts/Comments/Contributors?

q./

**Comment by compuguy** [comment]
I know that jitsi-meet isn't perfect, but it has a huge advantage that TokBox has: it works on a internal server (and doesn't need to talk to the internet for video/audio). Even Atlassian is using jitsi-meet as the basis of its video confrence/chat with Hipchat.

**Comment by designgurudotorg** [comment]
I agree that when it works JitsiMeet is pretty awesome @compuguy ...

**Comment by designgurudotorg** [comment]
If anyone wants to play with this: check out https://github.com/songz/cordova-plugin-opentok
